packages/utilities/__tests__/TextUtils.test.ts

```diff
@@ -62,4 +72,40 @@ describe("TextUtils", () => {
         const results = TextUtils.indentLines(text, "----");
         expect(results).toEqual(expected);
     });
+
+    it("should set chalk level to 0 if FORCE_COLOR is 0", () => {
+        process.env.FORCE_COLOR = "0";
+        const chalk = TextUtils.chalk;
+        expect(chalk.level).toEqual(0);
+    });
+
+    it("should set chalk level to 1 if FORCE_COLOR is 1", () => {
+        process.env.FORCE_COLOR = "1";
+        const chalk = TextUtils.chalk;
+        expect(chalk.level).toEqual(1);
+    });
+
+    it("should set chalk level to 2 if FORCE_COLOR is 2", () => {
+        process.env.FORCE_COLOR = "2";
+        const chalk = TextUtils.chalk;
+        expect(chalk.level).toEqual(2);
+    });
+
+    it("should set chalk level to 3 if FORCE_COLOR is 3", () => {
+        process.env.FORCE_COLOR = "3";
+        const chalk = TextUtils.chalk;
+        expect(chalk.level).toEqual(3);
+    });
+
+    it("should not set chalk level to 4 if FORCE_COLOR is 4", () => {
+        process.env.FORCE_COLOR = "4";
+        const chalk = TextUtils.chalk;
+        expect(chalk.level).not.toEqual(4);
+    });
+
+    it("should not set chalk level to fake if FORCE_COLOR is fake", () => {
+        process.env.FORCE_COLOR = "fake";
+        const chalk = TextUtils.chalk;
+        expect(chalk.level).not.toEqual("fake");
+    });
```

Comment:
For conciseness `TextUtils.chalk` can be used directly in the `expect` calls - but, no need for changes here especially since these are tests